pydundas/tests/rest/test_constant.py

```diff
@@ -0,0 +1,44 @@
+import unittest
+from unittest.mock import patch
+from pydundas import Api
+from pydundas.rest.constant import ConstantIdError, ConstantNameError, Constant
+
+
+class TestConstant(unittest.TestCase):
+
+    # Will be used to mock the actual constants.
+    mocked_constants = {
+            'NAME': 'id',
+            'NAME1': 'iddup',
+            'NAME2': 'iddup',
+        }
+
+    def test_get_by_valid_id_returns_good_name(self):
+        with patch.dict(Constant.constants, self.mocked_constants, clear=True):
```

Comment:
It's not a basic dictionary lookup as multiple values can be returned for one key. That is the whole point of those tests, independently from the rest. If/when I want to test the data, it will be another bunch of tests.